### PR TITLE
fix(platform): UI fixes; Fix block note UI displayed as normal block;Json-prettify execution output

### PR DESCRIPTION
**Pull request by majdyz**
### Background

Expected:
image

When it's executed:
image

### Changes 🏗️

* Re-purposed backend BlockUIType to generic BlockType.
* Avoid note block to be executed by the backend.
* Fix UI bug showing note block as a normal block.
* Prettify JSON output 
image

Only for the new autogpt platform, currently in autogpt_platform/

- Create from scratch and execute an agent with at least 3 blocks
- Import an agent from file upload, and confirm it executes correctly
- Upload agent to marketplace
- Import an agent from marketplace and confirm it executes correctly
- Edit an agent from monitor, and confirm it executes correctly

### CODE REVIEW

File: autogpt_platform/backend/backend/blocks/basic.py [hunk 1]
```diff
@@ -197,7 +191,7 @@ def __init__(self):
                 ("result", "Hello, World!"),
             ],
             categories={BlockCategory.INPUT, BlockCategory.BASIC},
-            ui_type=BlockUIType.INPUT,
+            block_type=BlockType.INPUT,
```
Comment (majdyz) [inline autogpt_platform/backend/backend/blocks/basic.py:194]: I'm repurposing `BlockUIType` to generic `BlockType`. So we can use it for other means not specific for UI on the backend.

File: autogpt_platform/frontend/src/components/DataTable.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default function DataTable({
                 {beautifyString(key)}
               </TableCell>
               <TableCell className="cursor-text">
-                <div className="flex min-h-9 items-center">
+                <div className="flex min-h-9 items-center whitespace-pre-wrap">
```
Comment (majdyz) [inline autogpt_platform/frontend/src/components/DataTable.tsx:54]: This is a cosmetic change, see the screenshot on the PR description.

File: autogpt_platform/frontend/src/components/RunnerUIWrapper.tsx [hunk 3]
```diff
@@ -31,12 +31,12 @@ const RunnerUIWrapper = forwardRef<RunnerUIWrapperRef, RunnerUIWrapperProps>(
     const getBlockInputsAndOutputs = useCallback(() => {
       const inputBlocks = filterBlocksByType(
         nodes,
-        (node) => node.data.block_id === "c0a8e994-ebf1-4a9c-a4d8-89d09c86741b",
+        (node) => node.data.uiType === BlockUIType.INPUT,
```
Comment (majdyz) [inline autogpt_platform/frontend/src/components/RunnerUIWrapper.tsx:34]: We already have uiType so we can utilize this to cleanup these kinds of logic.

**Comment by Swiftyos** [comment]
What was causing the note node to change to be rendered as a normal node?

**Comment by majdyz** [comment]
@Swiftyos the node is executed by the backend, it trigger an execution flow on the frontend that also triggers `setNodes`. And the nodes are not setting uiType: https://github.com/Significant-Gravitas/AutoGPT/pull/8169/files#diff-a3d14f257ccdf4e35bf653d076ab555d66c582f31d6704e736bfb2c903090548R155

**Review by Swiftyos — APPROVED** [review]
LGTM

**Pull request merged**